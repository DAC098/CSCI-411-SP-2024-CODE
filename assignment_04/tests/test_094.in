4
1 0 2
gtcc aaacaaggaagt
qeascodoadcwotiu ymeegzfplwleabzyyjitt
mrbxjensccryuhfetuxisqjjhiocoyyfzzlqem znpjnpatritzqhgjxqdwhoczarpwiwmk
mmscvgmfwvlkfmnefvivxtcxntcfocacdwpneg fqvpqboodeaimroiuuydznq
