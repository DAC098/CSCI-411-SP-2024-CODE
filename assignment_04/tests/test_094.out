__________gtcc
aaacaaggaagt__
10
___________qea_______scodoadcwotiu
ymeegzfplwl_eabzyyjit__________t__
18
___mrbxjen___sccr____yuh__fetuxisq__jjhiocoyyfz______zlqem_
znp____j_npat___ritzq__hgj____x__qdw__h_oc____zarpwiw____mk
21
mmscvgmf_wv______lkfmne_fvi__vxtcxntcfo____cacd_wpn_eg
_______fq_vpqbood_____ea__imr_________oiuuy___dz__nq__
16
